### ISSUE TITLE
I met an error when I build your project in Ubuntu

### ISSUE THREAD

**Issue by BITWN** [kaiyuzhao/XmdvTool#6]
I want to know why this error occurs and if you are so kind, do you know how to deal with it?
Here is the error:
release/.obj/Image.o: In function `Image::CreateTexture()':
Image.cpp:(.text+0x6b8): undefined reference to `gluBuild2DMipmaps'
release/.obj/GLCanvas.o: In function `GLCanvas::drawCircle(double, double, double, double)':
GLCanvas.cpp:(.text+0x4db): undefined reference to `gluNewQuadric'
GLCanvas.cpp:(.text+0x4eb): undefined reference to `gluQuadricDrawStyle'
GLCanvas.cpp:(.text+0x4f8): undefined reference to `gluQuadricNormals'
GLCanvas.cpp:(.text+0x551): undefined reference to `gluDisk'
release/.obj/GLCanvas.o: In function `GLCanvas::drawArc(double, double, double, double, double)':
GLCanvas.cpp:(.text+0x5a2): undefined reference to `gluNewQuadric'
GLCanvas.cpp:(.text+0x5b2): undefined reference to `gluQuadricDrawStyle'
GLCanvas.cpp:(.text+0x5bf): undefined reference to `gluQuadricNormals'
GLCanvas.cpp:(.text+0x61b): undefined reference to `gluPartialDisk'
release/.obj/GLCanvas.o: In function `GLCanvas::drawPartialDisk(double, double, double, double, double, double)':
GLCanvas.cpp:(.text+0x677): undefined reference to `gluNewQuadric'
GLCanvas.cpp:(.text+0x687): undefined reference to `gluQuadricDrawStyle'
GLCanvas.cpp:(.text+0x694): undefined reference to `gluQuadricNormals'
GLCanvas.cpp:(.text+0x6eb): undefined reference to `gluPartialDisk'
release/.obj/GLCanvas.o: In function `GLCanvas::mapScreenToData(Vec2 const&)':
GLCanvas.cpp:(.text+0x17a0): undefined reference to `gluUnProject'
release/.obj/GLCanvas.o: In function `GLCanvas::mapDataToScreen(Vec2 const&)':
GLCanvas.cpp:(.text+0x1880): undefined reference to `gluProject'
collect2: error: ld returned 1 exit status
Makefile.Release:817: recipe for target 'release/XmdvTool' failed
make: *** [release/XmdvTool] Error 1

**Comment by happysnackk** [comment]
Hi BITWN,
I ran into the same error, using linux mint 18 (which is a derivative of Ubuntu) with qt 5.
I found out that for some reason the -lGLU switch (necessary to include the library providing the glu-Functions) did not make it into the Makefile (lame pun intended). It is, however, specified in the XmdvTool.pro file, where there's a CXX_FLAGS section.

A quick fix was to directly append the -lGLU switch to the following line in the "Makefile" created by qmake:
LIBS = $(SUBLIBS) -L/usr/X11R6/lib64 -lQt5OpenGL -lQt5Widgets -lQt5Gui -lQt5Network -lQt5Xml -lQt5Core -lGL -lpthread

Also: using make -f Makefile.Debug or respectively Makefile.Release works on my system.
